source/simulator/core.cc

```diff
@@ -113,15 +113,15 @@ namespace aspect
      * computation.
      */
     template <int dim>
-    std_cxx11::unique_ptr<Mapping<dim> > construct_mapping(const GeometryModel::Interface<dim> &geometry_model,
-                                                           const InitialTopographyModel::Interface<dim> &initial_topography_model)
+    Mapping<dim> *construct_mapping(const GeometryModel::Interface<dim> &geometry_model,
+                                    const InitialTopographyModel::Interface<dim> &initial_topography_model)
```

Comment:
Ah, bummer. We went around this so many times to make the semantics obvious when @gassmoeller implemented it. It is so elegant with the unique-ptr...